scripts/setup/install_shared_libraries.bash

```diff
@@ -160,6 +160,20 @@ download_and_install_helib(){
     cd ..
 }
 
+download_and_install_seal(){
+    echo "Installing $SEAL..."
+    git clone https://github.com/SpiRITlab/SEAL.git $SEAL
+    cd $SEAL
+    git checkout master-SparkFHE
+    cd src
+    cmake -DCMAKE_CXX_COMPILER=g++-8 -DCMAKE_INSTALL_PREFIX=$libSparkFHE_root .
+    make CC=g++-8 LD=g++-8 LDLIBS+=-L$libSparkFHE_lib CFLAGS+=-I$libSparkFHE_include CFLAGS+=-fPIC
+    echo "Installing $SEAL... (DONE)"
```

Comment:
please confirm with MK, this only makes the project but not installing it.

Reply:
Worked with MK to modify these command, please test again.